src/Controller/SponsorshipBenefit/Delete.php

```diff
@@ -29,15 +30,15 @@ public function __construct(
         $this->csrfTokenManager = $csrfTokenManager;
     }
 
-    public function handle(Request $request, string $id): RedirectResponse
+    /**
+     * @param Request $request
+     * @param SponsorshipBenefit $sponsorshipBenefit
+     * @ParamConverter("sponsorshipBenefit", class="App:SponsorshipBenefit")
```

Comment:
Remove this annotation, you should be able to do it without expliciting the conversion.

Reply:
Then that would allow you to remove the docblock alltogether